code/src/components/MessageForm.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+const MessageForm =({onFormSubmit, newMessage, newMessageChange}) => {
+
+  return (
+    <section className="form-container">
+      <form onSubmit={ onFormSubmit}
+        className="form-box">
+        <label htmlFor="newMessage">What are you happy about today?</label>
+        <textarea
```

Comment:
When I write I message that is shorter than 5 characters or longer than 140 it creates a new message that says 'Could not save thought' and when I push the heart button it returns NaN. Not sure if this is on purpose. Or at least I'm having trouble finding the code that would make this 😂